projects/libusb.info/package.yml

```diff
@@ -0,0 +1,82 @@
+distributable:
+  url: https://github.com/libusb/libusb/releases/download/v{{ version }}/libusb-{{ version }}.tar.bz2
+  strip-components: 1
+
+versions:
+  github: libusb/libusb/releases/tags 
+  strip: /^v/
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: ^2
+  script: |
+    ./configure $ARGS
+    make install
+  env:
+    ARGS:
+      - --prefix="{{prefix}}"
+      - --disable-dependency-tracking
+      - --disable-debug
+
+test:
+  dependencies:
+    tea.xyz/gx/cc: c99 
+  script: |
+    mv $FIXTURE test.c
+    gcc test.c -lusb-1.0 "-I{{ prefix }}/include/libusb-1.0" -o test
```

Comment:
Instead of the `-I` flag, we should prefer this on the build script:

```
cd "{{prefix}}/lib"
mv libusb-1.0/* .
rmdir libusb-1.0
ln -s . libusb-1.0
```